aptos-move/transaction-replay/src/main.rs

```diff
@@ -18,9 +18,6 @@ struct Opt {
     /// Path to the local AptosDB file
     #[structopt(long, parse(from_os_str))]
     db: Option<PathBuf>,
-    /// Full URL address to connect to - should include port number, if applicable
```

Comment:
Lol why are you updating this?

Reply:
unused, apparently the dead code detection is much better